lib/ajValidation/ajvValidation.js

```diff
@@ -97,6 +136,11 @@ function validateSchema (schema, valueToUse, options = {}, jsonSchemaDialect) {
       isPmVariable(dataPath === '' ? valueToUse : _.get(valueToUse, dataPath))) {
       return false;
     }
+
+    if (validationError.keyword === 'type') {
+      return !isTypeValue(_.get(valueToUse, dataPath), validationError.params.type);
```

Comment:
@LuisTejedaS Is it always guaranteed that `params.type` is present in validation error reported by Ajv.